File: QuantConnect.AlphaStream/Models/Conversation.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using QuantConnect.AlphaStream.Infrastructure;

namespace QuantConnect.AlphaStream.Models
{
    public class Conversation
    {
        // <summary>
        /// Subject line of the message
        /// </summary>
        [JsonProperty("subject")]
        public string Subject { get; set; }

        /// <summary>
        /// Contents of the message
        /// </summary>
        [JsonProperty("message")]
        public string Message { get; set; }

        /// <summary>
        /// Dictionary of message sender information keyed by the type of information.
        /// If the key is 'id', From returns the sender anonymized hash id.
        /// If the key is "type", From returns the type of the sender, either "client" or "author"
        /// </summary>
        [JsonProperty("from")]
        public Dictionary<string, string> From { get; set; }

        /// <summary>
        /// UTC time message was sent
        /// </summary>
        [JsonProperty("timestamp"), JsonConverter(typeof(DoubleUnixSecondsDateTimeJsonConverter))]
        public DateTime UtcTimeReceived { get; set; }
    }
}

File: AlphaStream/Models/Conversation.py
from datetime import datetime

class Conversation:
    """ Conversation created between a client and an author """
    def __init__(self, result):
        self.Subject = result.get('subject', None)
        self.From = result.get('from', None)
        self.Message = result.get('message', None)
        self.TimeReceived = datetime.utcfromtimestamp(result.get('timestamp', None)) if result.get('timestamp', None) is not None else None

    def __repr__(self):
        return f'Conversation from {self.From["id"]} at {self.TimeReceived}. Subject: {self.Subject}.  ::  Message: {self.Message}'